### PR TITLE
Add support for releasing editions

### LINKED ISSUE #26: Script to make editioning repeatable
We should script the process of archiving the current edition and publishing a new edition to prevent errors like TeamBasedInquiryLearning/linear-algebra#339 from slipping by.  Some notes from that discussion on what this script should do:

> To archive a finalized edition and create a new preview edition:
>
>Update deploy-dirs at https://github.com/TeamBasedInquiryLearning/linear-algebra/blob/main/project.ptx
>
>Update subtitles at https://github.com/TeamBasedInquiryLearning/linear-algebra/blob/main/source/main.ptx
>
>Update title at https://github.com/TeamBasedInquiryLearning/linear-algebra/blob/main/exercises/bank.xml
>
>Archive finalized edition at https://github.com/TeamBasedInquiryLearning/linear-algebra/tree/main/site
>
>Probably other stuff I'm forgetting because I haven't automated this yet, in the hopes that CLI adds built-in version support. >(Though it probably wouldn't also support all the extra CheckIt stuff anyway I guess.)

I'll add:

- Update any CheckIt links in the text
- Update landing page

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #26

### CODE REVIEW

File: source/precalculus/source/meta/frontmatter.ptx [hunk 1]
```diff
@@ -70,6 +70,28 @@ This work includes materials used under license from the following works:
...
+We're providing this first edition for those who are eager to try out
+TBIL in their precalculus classrooms, with the understanding that this
+book will likely change a lot between now and our next edition, once we've
+tried it in our own classrooms! Are you interested in joining the community
+using these materials, and possibly contributing back to these free and
+open-source resources? Join the conversation at <url href="http://chat.tbil.org"/>;
+we look forward to hearing from you.
+      </p>
+  
+    </paragraphs>
```
Comment (StevenClontz) [inline source/precalculus/source/meta/frontmatter.ptx:93]: @siwelwerd please review/wordsmith

File: source/precalculus/source/meta/frontmatter.ptx [hunk 2]
```diff
@@ -70,6 +70,28 @@ This work includes materials used under license from the following works:
...
     <title>For Instructors</title>
+    <paragraphs xml:id="features">
+      <title>Welcome to our first edition!</title>
+   
+      <p>
+We're excited to share with you our first edition of
+Precalculus for Team-Based Inquiry Learning! These activities, which mostly
+cover topics from College Algebra, are being
+used in a few classrooms for the first time in Fall 2024. More
+activities are being designed to be included in our 2025 editions.
```
Comment (siwelwerd) [inline source/precalculus/source/meta/frontmatter.ptx:81]:
```suggestion
activities are being designed to be included in our 2025 editions,
with a goal of fully supporting Precalculus courses in Fall 2025.
```

File: source/precalculus/source/meta/frontmatter.ptx [hunk 3]
```diff
@@ -70,6 +70,28 @@ This work includes materials used under license from the following works:
...
+      </p>
+      <p>
+We're providing this first edition for those who are eager to try out
+TBIL in their precalculus classrooms, with the understanding that this
+book will likely change a lot between now and our next edition, once we've
+tried it in our own classrooms! Are you interested in joining the community
+using these materials, and possibly contributing back to these free and
+open-source resources? Join the conversation at <url href="http://chat.tbil.org"/>;
+we look forward to hearing from you.
```
Comment (siwelwerd) [inline source/precalculus/source/meta/frontmatter.ptx:90]:
```suggestion
we look forward to hearing from you, particularly if you have feedback
from your students on these activities.
```

File: source/precalculus/source/meta/frontmatter.ptx (hunk 1 above)
Comment (siwelwerd) [reply source/precalculus/source/meta/frontmatter.ptx:93]: Looks good, two minor suggestions made

File: source/precalculus/source/meta/frontmatter.ptx [hunk 4]
```diff
@@ -70,6 +70,28 @@ This work includes materials used under license from the following works:
 
   <preface xml:id="instructor-notes">
     <title>For Instructors</title>
+    <paragraphs xml:id="features">
```
Comment (StevenClontz) [inline source/precalculus/source/meta/frontmatter.ptx:73]:
```suggestion
    <paragraphs xml:id="first-edition-welcome">
```

**Comment by StevenClontz** [comment]
This is close-enough to working; I want the actual publication of 2024 editions in a clean PR, so merging.

**Pull request merged**